config/database.yml

```diff
@@ -0,0 +1,21 @@
+# Use this example code to connect the project with your database
+# You will need to create a file named <database.yml> with this code (or modify the name of this file)
+# also you will need to add or edit the values in <.env.exameple>
+default: &default
+  adapter: postgresql
+  user: <%= ENV['DATABASE_USER'] %>
+  password: <%= ENV['DATABASE_PW'] %>
+  host: <%= ENV['DATABASE_HOST'] %>
+  port: <%= ENV['DATABASE_PORT'] %>
+  encoding: unicode
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+
+development:
+  <<: *default
+  database: <%= ENV['DATABASE_DEVELOPMENT'] %>
+test:
+  <<: *default
+  database: <%= ENV['DATABASE_TEST'] %>
+
+production:
+  database: <%= ENV['DATABASE_URL'] %>
```

Comment:
You should not upload this file to GitHub. Instead, you must put an example file: database.yml.example

Reply:
We need this file in repo, because we can't create files en heroku, and we need to take values from Heroku envs config variables.

Reply:
Fixed! added again to .gitignore

Reply:
@HeikelV, The database.yml file still appears on GitHub, I recommend that you use this command to make it disappear.

`git rm --cached config/database.yml`

Reply:
ready, file was deleted